docs/getting-started/csm-stack/image-to-ground-tutorial.ipynb

```diff
@@ -13,10 +13,19 @@
    "id": "3602c014-53bc-4330-a9b0-0848d4927458",
    "metadata": {},
    "source": [
-    "Lessons learned in this tutorial: \n",
-    "* How to generate Image Support Data (ISD) for an image\n",
-    "* instantiate a CSM camera model\n",
-    "* Perform a simple ground to image call \n",
+    "Goals in this tutorial:\n",
+    "\n",
+    "- [ ] Generate Image Support Data (ISD) for an image\n",
+    "- [ ] Instantiate a CSM camera model\n",
+    "- [ ] Perform a simple ground to image call \n",
```

Comment:
for aesthetic reasons, maybe make these rechecked since it looks better, idk if users would ever actually use them to track progress?

Reply:
If you don't think people will use the checkboxes, I'll just leave them as plain bullet points